week2/homework/src/index.js

```diff
@@ -1,3 +1,85 @@
 'use strict';
+const fs = require('fs');
+const DEFAULT_ENCODING = 'utf8';
+const FILE_PATH = 'c:/nodeJS/week2/homework/src/todo.txt';
```

Comment:
this will not work on all devices, but windows only and only if the user have the project in that exact location on there devices!

instead you do the following:
```suggestion
const FILE_PATH = __dirname + '/todo.txt';
```
this should work just fine in any platform.

<img width="871" alt="Screen Shot 2019-08-29 at 15 25 35" src="https://user-images.githubusercontent.com/25272822/63944758-70db1800-ca72-11e9-8275-075bbe1a2298.png">

Reply:
ok. So __dirname defines file path globally. So it works all other devices. Not only speciliazed for my computer?

Reply:
yup!